src/composites/FeaturesContainer/Overview/OverviewDropLists.tsx

```diff
@@ -0,0 +1,81 @@
+// import { Component, ReactNode } from "react";
+import { overviewData, listNode } from './index.ts';
+import * as Accordion from "@radix-ui/react-accordion";
+import { Button, Theme, Box, Text, Strong } from '@radix-ui/themes';
+import { DotFilledIcon } from '@radix-ui/react-icons';
+import "./Overview.css"
+
+export function OverviewDropList(risk:string, section:string, props:overviewData): JSX.Element {
+
+    /* Creates a new array of all nodes matching the given classification and risk level */
+    const targetArray: listNode[] = getRelatedArray(section, risk, props);
+
+    /* Finds suitable class name for css styling */
+    const childClass: string = getChildClass(risk);
+
+    if (targetArray.length === 0) {
+        /* Returns a paragraph in place of a descriptive list */
+        return ( 
+            <Accordion.Item value = 'noNodes'>
+                No nodes match this risk level
+            </Accordion.Item>
+        );
+    }
+    else {
+        const items = targetArray.map((item, index:number) => (
+            <Box className = {childClass} key={index}>
+                <Text as='p'><Strong>{item.name}</Strong>
+                    { /* Zooms tree graph to given node */ }
+                    <Button className='Button'>View Node</Button>
+                </Text>
+                <Text as='p'><DotFilledIcon/>{"Value: " + item.value}</Text>
+                {item.description && (<Text as='p'><DotFilledIcon/>{"Description: " + item.description}</Text>)}
+                <Text as='p'><DotFilledIcon/>{"Evaluation Strategy: " + item.eval_strategy}</Text>
+                <Text as='p'><DotFilledIcon/>{"Normalizer: " + item.normalizer}</Text>
+                <Text as='p'><DotFilledIcon/>{"Utility Function: " + item.utility_function}</Text>
+            </Box>
+        ));
+        return (
+            <Box>
+                <Theme
+                    accentColor="gray"
+                    grayColor="gray"
+                    panelBackground="solid"
+                    scaling="100%"
+                    radius="full">
+                    {items}
+                </Theme>
+            </Box>
+        );
+    }
+}
+
+function getChildClass(riskLvl: string): string
+{
+    switch (riskLvl){
```

Comment:
You can also create a dictionary and achieve this:
```javascript
const mapping = {severe: 'SevereTitleCard', ...}
if(riskLevel in mapping) return mapping[riskLevel]
// handle errors

```

Reply:
This function has been removed due to a small restructuring.